File: Elixir/Iniciante/1008.exs
n = IO.gets("") |> String.trim() |> String.to_integer
n_horas = IO.gets("") |> String.trim() |> String.to_integer
valor_horas = IO.gets("") |> String.trim() |> String.to_float

salario = n_horas * valor_horas

IO.puts "NUMBER = #{n}"
IO.puts "SALARY = U$ #{Float.to_string(salario, decimals: 2)}"

File: GO/Iniciante/1008.go
package main

import (
	"fmt"
)

func main() {

  var n int
  var horas, salario float64

  fmt.Scan(&n)
  fmt.Scan(&horas)
  fmt.Scan(&salario)

  salario *= horas
  
  fmt.Printf("NUMBER = %d\n", n)
  fmt.Printf("SALARY = U$ %.2f\n", salario)
  
}
